source/tutorials/nvidia.rst

```diff
@@ -146,12 +160,19 @@ Install the NVIDIA drivers
    installation progress. Switch to a secondary virtual terminal by pushing
    :command:`CTRL + ALT + F2` or remotely login over SSH.
 
-#. Navigate to the directory where the NVIDIA installer was downloaded.
+#. Navigate to the directory where the NVIDIA installer was downloaded. In
+   this example, it was saved in the :file:`Downloads` folder.
 
    .. code-block:: bash
 
       cd ~/Downloads/
 
+#. Make the :file:`NVIDIA-Linux-x86_64-<VERSION>.run` file executable.
+
+   .. code-block:: bash
+
+      chmod +x :file:`NVIDIA-Linux-x86_64-<VERSION>.run`   
```

Comment:
In this case, please remove Sphinx syntax of `:file:`

Reply:
Oops. Thanks.